### PR TITLE
[DDW-148] fix merge conflicts with release/0.9.0

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR merges in the `release/0.9.0` into the fork and fixes all merge conflicts. This should also include all the devops changes and stuff that has been done outside of PRs 😉

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -1,10 +1,24 @@
+<<<<<<< HEAD
```
Comment (nikolaglumac) [inline default.nix]: @DominikGuzei seems like this file contains conflicts? :(

**Comment by nikolaglumac** [comment]
Reviewing this one now.

**Comment by nikolaglumac** [comment]
@DominikGuzei watch feature works now 🎉 
Thanks!

**Comment by DominikGuzei** [comment]
@nikolaglumac i fixed all the CI issues that were obviously caused by our code … the rest has to be fixed by devops. Will merge this now! 🎉

**Pull request merged**